setup.py

```diff
@@ -123,7 +123,7 @@ def build_extensions(self):
 
         opts = []
         if sys.platform == "darwin":
-            opts += ['-stdlib=libc++', '-mmacosx-version-min=10.7']
```

Comment:
Are you sure about this? I suspect it is needed in case clang is used as the compiler.

Reply:
Clang will be a pain in the ass with OpenMP, in my opinion.
At least, I never managed to make clang work for `adcc`.
I removed the option because gcc fails otherwise.

Reply:
There is a remedy for that, use `has_flag` and only add the flag it the answer is true.

Reply:
Resolved.